docs/resource/process.md

```diff
@@ -156,17 +156,22 @@ Examples for some JavaScript runtimes
 
 ### .NET Runtimes
 
-TODO(<https://github.com/open-telemetry/opentelemetry-dotnet/issues/1281>): Confirm the contents here
+.NET instrumentation SHOULD fill in the values by following values:
 
-| Value | Description |
-| --- | --- |
-| `dotnet-core` | .NET Core, .NET 5+ |
-| `dotnet-framework` | .NET Framework |
-| `mono` | Mono |
+- `process.runtime.name` - Fill in the value by the name of runtime.
```

Comment:
What values are to be expected here? The values listed [here](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.frameworkdescription?view=net-6.0#remarks)?

Reply:
Yes, at least for .NET and .NET Framework.
There is no plan to support .NET Core.

Reply:
Maybe then we can add some more guidance? For example the one for `process.runtime.version` has a clear way of doing it. If the same is true for runtime name we should add.

Reply:
By convention, we could take:
"Use name part from `System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription`". But it requires parsing. I would like to avoid it when possible.

Reply:
Could we then at least give a list of possible values (only ".NET" and ".NET Framework", if I understand correctly)?

Otherwise, values like "dotnet-core" and "dotnet-framework" would still be compliant. And I think that's not what is desired.

Reply:
@pyohannes, fixed in aee236b106c3e19b088021f003b1be17c77427c1. Please let me know if you are fine with the wording.